portafly/.stylelintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "stylelint-config-sass-guidelines"
+  "extends": "stylelint-config-sass-guidelines",
+  "rules": {
+    "selector-class-pattern": null
```

Comment:
This is to fix the error: https://github.com/3scale/porta/pull/1889/files#diff-94b2cd13981c12108120c7f72bdbe5c3L1